lib/API/Startup.js

```diff
@@ -89,51 +90,51 @@ module.exports = function(CLI) {
       return cb();
     }
 
-    if (fs.existsSync('/etc/init.d/pm2-init.sh')) {
+    if (fs.existsSync('/etc/init.d/' + service_name + '-init.sh')) {
```

Comment:
Rollback this change, this is used for backward compatibility cause the old startup system had only this file for every platform.